Makefile

```diff
@@ -35,7 +35,7 @@ ifeq ($(OS),Windows_NT)
 SHELL=cmd
 WINDOWS_IPATHS=-I./cache/include
 WINDOWS_OPT=-LD -EHsc -Feluasteam
```

Comment:
`CONTRIBUTING.md` specifies VS 2017, so I figure it makes sense to use the path from that in the Makefile. I think this is helpful for people like me who don't build things under Windows very often.

Reply:
As long as this works on Travis, I'm happy with it. Seems the tests on your PR didn't work (maybe because of some secrets, I'm not quite sure, seems to be working on a [new branch](https://travis-ci.org/github/uspgamedev/luasteam/builds/747978905) I pushed with these changes).

Reply:
Oh yeah. I feel silly, the fact that was being used for automated builds slipped my mind.

I'll revert this, it was just a little idea but I think most people are capable of finding `vcvarsall` themselves.